compiler/src/dotty/tools/dotc/quoted/QuoteDriver.scala

```diff
@@ -61,7 +62,15 @@ class QuoteDriver extends Driver {
 
   override def initCtx: Context = {
     val ictx = super.initCtx.fresh
-    val classpath = System.getProperty("java.class.path")
```

Comment:
What about using ctx.settings.classpath instead, isn't that enough?

Reply:
No, the classpath of the library in not in there. That is why I needed to add it explicitly. We also need the current classpath in case the quote refers to something in the application (such as class or static method).

Reply:
For example
```
class Foo
object Foo {
  def main(...) = {
    val q = '(new Foo)
    q.run // Foo need to be present in the classpath of the compiler that will compile q
  }
```

Reply:
In scala.reflect macros, it assumes separation (ahead-of) compilation of macro definitions. The user-app must depend on the lib that contains `Foo` in order to use that macro, thus it's automatically on the path.